src/utils/Shell.ts

```diff
@@ -28,6 +34,27 @@ class Bash extends Shell {
     get preCommandModifiers(): string[] {
         return [];
     }
+
+    loadHistory(): { lastModified: Date, commands: string[] } {
+        if (process.env.HISTTIMEFORMAT) {
```

Comment:
This check is pretty much useless: Black Screen will only pick up shell's environment if it's started from a shell session. But usually it's started as a standalone application.